app-connector/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:11-alpine as builder

```

Comment:
We don't need a docker image for the library for now, please remove

Reply:
removed image